### PR TITLE
Create AT1147CaliphAtrib.xml

### PR DESCRIPTION
**Pull request by GueshSol**
https://github.com/BetaMasaheft/Documentation/issues/2362

### CODE REVIEW

File: new/AT1147CaliphAtrib.xml [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?>
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="AT1147CaliphAtrib" xml:lang="en" type="auth">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>A Caliph from Ātrib tries to destroy a Church</title>
```
Comment (thea-m) [inline new/AT1147CaliphAtrib.xml]: ʾAtrib

File: new/AT1147CaliphAtrib.xml [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p>A miniature with four episodes, showing: first, the Caliph ordering his troops to
+                    destroy a Church in Ātrib; second, <persName ref="PRS6819Mary"/>"/> appearing to the
```
Comment (thea-m) [inline new/AT1147CaliphAtrib.xml]: ʾAtrib

Comment (thea-m) [inline new/AT1147CaliphAtrib.xml]: `<persName ref="PRS6819Mary"/>"/>` - here there is one `"/>` too much

File: new/AT1147CaliphAtrib.xml [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+                    Caliph (who was in sleep) to order his agent to stop the destruction of the
```
Comment (thea-m) [inline new/AT1147CaliphAtrib.xml]: "who was asleep" would be better, I think

File: new/AT1147CaliphAtrib.xml [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+                    Church with a white dove appearing to take his letter between them; third, the
+                    dove giving the letter written by the caliph to the head of troops (agent of
+                    the Caliph) with the Abbot of the Church standing in the gate, and
+                    fourth, the Caliph who later become believer sitting in front of the Church.</p>
```
Comment (thea-m) [inline new/AT1147CaliphAtrib.xml]: "who later became a believer"

**Review by thea-m — CHANGES_REQUESTED** [review]
Very good! I just added some suggestions

**Comment by thea-m** [comment]
@GueshSol Could you please carry out the corrections in the file, so that this can be merged? Thanks a lot!

**Comment by eu-genia** [comment]
@GueshSol

**Comment by GueshSol** [comment]
@thea-m there seems deletion with the branch under discussion. I was trying to carry out the corrections you provided. The art ID does not appear in the art theme section or even in  the new sub-repository. I have the branch on my GitHub not deleted even after several months, but
 the branch is not in the list. What should I do?

**Comment by eu-genia** [comment]
You have to switch to the branch you were working in in your GitHub desktop.
As long as the branch has not been merged you cannot see the record in the master branch.

**Comment by eu-genia** [comment]
The name of the branch is `Guesh-26-23` as you can see above

**Comment by eu-genia** [comment]
Dear Guesh, please switch to this branch and carry out your changes here, I will close the new pull request

**Comment by eu-genia** [comment]
image

File: new/AT1147CaliphAtrib.xml [hunk 5]
```diff
@@ -56,6 +56,7 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
         </profileDesc>
         <revisionDesc>
             <change who="GS" when="2023-05-25">Created
+              <change who="GS" when="2024-09-12">corrected typo errors
```
Comment (eu-genia) [inline new/AT1147CaliphAtrib.xml]: please check the nesting, you inserted change inside of another change

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by GueshSol** [comment]
I see. I think I solved it. Thank you.

File: new/AT1147CaliphAtrib.xml (hunk 4 above)
Comment (eu-genia) [reply new/AT1147CaliphAtrib.xml]: this has not been corrected

File: new/AT1147CaliphAtrib.xml [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+                    <term key="Apocrypha"/>
+                    <term key="Miracle"/>
+                    <term key="Hagiography"/>
+                    <term key="ChristianContent"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            <langUsage>
```
Comment (eu-genia) [inline new/AT1147CaliphAtrib.xml]: always before submitting anything check in your oxygen that the document is valid (green). This document is not well formed XML
in line 55 you need the closing tag `</langUsage>`, not the opening tag

**Review by eu-genia — CHANGES_REQUESTED** [review]

**Comment by GueshSol** [comment]
I know what you said. Oxygen gives you a chance if any element is misplaced, wrongly added or omitted but for this art theme, my Oxygen application did not help me to edit the branch. I tried to fix it using an Atom app that automatically give me a chance to edit it. That's why I am unable to see some of the minor errors; I did not manage it well working using Atom for XML description.  It should work now.

**Review by eu-genia — APPROVED** [review]

**Comment by eu-genia** [comment]
I believe this PR has not been merged, what is the reason you are closing an unmerged pull request?

**Comment by eu-genia** [comment]
It was waiting to be approved by @thea-m in order to be merged.

**Comment by GueshSol** [comment]
@eu-genia  I apologize for the error; I just thought it was done.

**Review by thea-m — APPROVED** [review]

**Pull request merged**